doc/installation.rst

```diff
@@ -14,6 +14,26 @@ Thanks!
 
 
 
+
+=================
+Spyder Installers
+=================
+
+Here you will find our standalone installers for Windows and macOS which are avaliable from Spyder 4.2 onwards. We recommend using this installation method, but if you want to try another or are using Linux, go to the sections below.
```

Comment:
```suggestion
Our standalone installers for Windows and macOS are available from Spyder 4.2 onwards.
We recommend using this installation method on those platforms, but we offer several other options for Linux, advanced users and specific needs, so keep reading if that's the case for you.
```

* Sentence breaks
* Fixed spelling mistake
* Simplify and de-redundantize first sentence
* Use friendlier, more specific and more engaging language in second sentance